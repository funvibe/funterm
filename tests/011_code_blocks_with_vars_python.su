py (greeting, message) {
    def greeting(hello):
        print(hello)
    message = "hello again"
}
py.greeting("hello from python")
py.print(py.message)