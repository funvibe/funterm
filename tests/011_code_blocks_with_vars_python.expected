hello from python
hello again